assets/base/i18n/en.json

```diff
@@ -348,34 +347,50 @@
-  "mobile.about.serverVersion": "Server Version: {version} (Build {number})",
-  "mobile.about.serverVersionNoBuild": "Server Version: {version}",
   "mobile.account.settings.save": "Save",
   "mobile.action_menu.select": "Select an option",
   "mobile.add_team.create_team": "Create a new team",
   "mobile.add_team.join_team": "Join Another Team",
   "mobile.android.back_handler_exit": "Press back again to exit",
   "mobile.android.photos_permission_denied_description": "Upload photos to your server or save them to your device. Open Settings to grant {applicationName} Read and Write access to your photo library.",
   "mobile.android.photos_permission_denied_title": "{applicationName} would like to access your photos",
+  "mobile.calls_call_ended": "Call ended",
   "mobile.calls_call_screen": "Call",
+  "mobile.calls_chat_thread": "Chat thread",
+  "mobile.calls_current_call": "Current call",
   "mobile.calls_disable": "Disable Calls",
   "mobile.calls_enable": "Enable Calls",
   "mobile.calls_end_call_title": "End call",
   "mobile.calls_end_msg_channel": "Are you sure you want to end a call with {numParticipants} participants in {displayName}?",
+  "mobile.calls_end_msg_channel_default": "Are you sure you want to end the call?",
   "mobile.calls_end_msg_dm": "Are you sure you want to end a call with {displayName}?",
   "mobile.calls_end_permission_msg": "You do not have permission to end the call. Please ask the call creator to end the call.",
   "mobile.calls_end_permission_title": "Error",
+  "mobile.calls_ended_at": "Ended at",
   "mobile.calls_error_message": "Error: {error}",
   "mobile.calls_error_permissions": "no permissions to microphone, unable to start call",
   "mobile.calls_error_title": "Error",
   "mobile.calls_join_call": "Join Call",
+  "mobile.calls_lasted": "Lasted {duration}",
+  "mobile.calls_leave": "Leave",
   "mobile.calls_leave_call": "Leave Call",
+  "mobile.calls_lower_hand": "Lower hand",
+  "mobile.calls_more": "More",
+  "mobile.calls_mute": "Mute",
+  "mobile.calls_name_is_talking": "{name} is talking",
+  "mobile.calls_name_started_call": "{name} started a call",
+  "mobile.calls_noone_talking": "No one is talking",
   "mobile.calls_not_available_msg": "Please contact your system administrator to enable the feature.",
   "mobile.calls_not_available_option": "(Not Available)",
   "mobile.calls_not_available_title": "Calls is not enabled",
   "mobile.calls_ok": "OK",
+  "mobile.calls_raise_hand": "Raise hand",
   "mobile.calls_see_logs": "see server logs",
+  "mobile.calls_speaker": "Speaker",
   "mobile.calls_start_call": "Start Call",
+  "mobile.calls_unmute": "Unmute",
+  "mobile.calls_viewing_screen": "You are viewing {name}'s screen",
+  "mobile.calls_you": "(you)",
   "mobile.camera_photo_permission_denied_description": "Take photos and upload them to your server or save them to your device. Open Settings to grant {applicationName} Read and Write access to your camera.",
```

Comment:
```suggestion
  "mobile.camera_photo_permission_denied_description": "Take photos and upload them to your server or save them to your device. Open Settings to grant {applicationName} read and write access to your camera.",
```

Reply:
All changes must happen in both, this file and in the code. So apply the suggestions being mundful of that